.github/workflows/cron-weekly.yml

```diff
@@ -8,33 +8,33 @@ jobs:
   check-links:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - uses: gaurav-nelson/github-action-markdown-link-check@a996638015fbc9ef96beef1a41bbad7df8e06154
         # checks all markdown files from /docs including all subfolders
         with:
-          use-quiet-mode: 'yes'
-          use-verbose-mode: 'yes'
-          config-file: '.github/workflows/markdown.links.config.json'
-          folder-path: 'docs/'
-      - uses: actions/checkout@v3
+          use-quiet-mode: "yes"
```

Comment:
Does this still work without the quotes/whitespace changes? If so can we reduce churn by just upgrading the `v3` to `v4`

Reply:
I'll take a look.

Reply:
It looks like this still works without the formatting changes.